tests/test_types_generator.py

```diff
@@ -170,8 +170,12 @@ def bucket_data_tagging(expiry: str = "unset", sensitivity: str = "unset") -> Bu
     return BucketDataTagging(expiry=expiry, sensitivity=sensitivity)
 
 
-def bucket_encryption(enabled: bool = False, type: Optional[str] = None) -> BucketEncryption:
-    return BucketEncryption(enabled=enabled, type=type)
+def bucket_encryption(enabled: bool = False, key: str = "", type: Optional[str] = None) -> BucketEncryption:
```

Comment:
If `key` is made optional in the `BucketEncryption` type, the `key` function arg here should reflect that change too.